### PR TITLE
Correct styling and spelling

### PR DESCRIPTION
**Pull request by karl-johan-grahn**

### CODE REVIEW

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-89-216f1927`

**Review by usamaahmadkhan — DISMISSED** [review]

File: docs/content/sre/machinepools/machinepools.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ The overall minimum resource requirements are:
 | Worker | 4 x 16 x 100 | 3 | 12 | 48 | 300 | Yes |
 | **Total minimum** | | 9 | 36 | 208 | 900 | |
 | **Total recommended** | | 11 | 44 | 240 | 1100 | |
+
```
Comment (karl-johan-grahn) [inline docs/content/sre/machinepools/machinepools.md:25]: without this break the heading is not displayed but meshed together into the table

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-89-9939c128`

**Review by usamaahmadkhan — DISMISSED** [review]

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-89-f1f40acd`

**Review by usamaahmadkhan — APPROVED** [review]

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.60`

**Pull request merged**